homeassistant/components/syncthru/config_flow.py

```diff
@@ -0,0 +1,128 @@
+"""Config flow for Samsung SyncThru."""
+
+import re
+from urllib.parse import urlparse
+
+from pysyncthru import SyncThru
+from url_normalize import url_normalize
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import ssdp
+from homeassistant.const import CONF_NAME, CONF_URL
+from homeassistant.helpers import aiohttp_client
+
+# pylint: disable=unused-import # for DOMAIN https://github.com/PyCQA/pylint/issues/3202
+from .const import DEFAULT_MODEL, DEFAULT_NAME_TEMPLATE, DOMAIN
+
+
+class SyncThruConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Samsung SyncThru config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    async def async_step_user(self, user_input=None):
+        """Handle user initiated flow."""
+        if user_input is None:
+            return await self._async_show_form(step_id="user")
+        return await self._async_check_and_create("user", user_input)
+
+    async def async_step_import(self, user_input=None):
```

Comment:
There is no code to initialize `import` so you can remove this.

Reply:
I thought it was invoked by `async_init(..., context={"source": SOURCE_IMPORT}, ...)` in sensor, am I mistaken?

Reply:
You are right. Sorry, I missed that.